tools/ci-cdk/canary-lambda/README.md

```diff
@@ -0,0 +1,32 @@
+Canary Lambda
+=============
+
+This is a Rust Lambda that exercises the AWS SDK for Rust in a real Lambda
```

Comment:
is there a one shot command to build/deploy with CDK? can you include that here if there is?

Reply:
IIRC, CDK doesn't provide this directly, but you can add a target to the `package.json` that does the build for you, and have the CDK synth or deploy depend on that target.